plugins/orchestrator-backend/src/service/api/mapping/V2Mappings.ts

```diff
@@ -158,3 +162,18 @@ export function mapToGetWorkflowInstanceResults(
 
   return returnObject;
 }
+
+export function mapToWorkflowRunStatusDTO(
+  status: ProcessInstanceState,
+): WorkflowRunStatusDTO {
+  return {
+    key: firstLetterToUppercase(status),
+    value: status,
+  };
+}
+
+export function firstLetterToUppercase(text: string): string {
+  if (text === undefined || text === null || text.length < 1) return text;
+
+  return text[0].toUpperCase() + text.slice(1).toLowerCase();
+}
```

Comment:
We have this [file](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/orchestrator/src/utils/StringUtils.ts) in the FE. Can you please move it to the commons package and reuse the functions here?